core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycleSpec.scala

```diff
@@ -373,21 +432,23 @@ class BpmBillingProjectLifecycleSpec extends AnyFlatSpec {
     val bp =
       new BpmBillingProjectLifecycle(repo, bpm, workspaceManagerDAO, mock[WorkspaceManagerResourceMonitorRecordDao])
 
-    val result = bp.preDeletionSteps(
-      billingProjectName,
-      testContext
+    Await.result(bp.preDeletionSteps(
```

Comment:
This is the test case related to the change in behavior-- we will still delete the billing profile in this case and just log the issue with deleting the landing zone.